src/test/java/codeu/model/store/basic/UserStoreTest.java

```diff
@@ -84,6 +85,55 @@ public void testIsUserRegistered_false() {
     Assert.assertFalse(userStore.isUserRegistered("fake username"));
   }
 
+  @Test
+  public void testUpdateUserDescription_userNotRegistered() {
+    User fakeUser =
+        new User(
+            UUID.randomUUID(),
+            "test_username_three",
+            "password",
+            Instant.now());
+    String newDescription = "new description";
+    userStore = Mockito.mock(UserStore.class);
+    Mockito.when(userStore.isUserRegistered(fakeUser.getName()))
+        .thenReturn(false);
+    Mockito.when(userStore.updateUserDescription(
+        fakeUser,
+        newDescription))
+        .thenCallRealMethod();
+    Assert.assertFalse(userStore.updateUserDescription(
+        fakeUser,
+        newDescription));
+  }
+
+  @Test
+  public void testUpdateUserDescription_updateSucessful() {
+    String newDescription = "new description";
+    Mockito.when(
+        mockPersistentStorageAgent.updateUserDescription(USER_ONE.getName(), newDescription))
```

Comment:
just for my own understanding: What is USER_ONE?

Reply:
fake user object defined in UserStoreTest.java:20
In intellij, you can cmd+click or cmd+b on a symbol to go to its declaration.